docrane/util.py

```diff
@@ -100,7 +102,8 @@ def convert_params(params):
                 converted_params[param] = ast.literal_eval(
                     params.get(param))
             except (ValueError, SyntaxError):
-                LOG.error("Malformed param '%s'. Skipping..." % param)
+                LOG.error("Possible malformed param '%s'." % param)
```

Comment:
It will keep them as strings so docker can attempt to parse them as such. I
ran into a weird issue in one case where command can be either a string or
a list, I set it to be a string with some special chars (&), and ast
crapped the bed trying to do something with it.